app/models/manageiq/providers/openstack/cinder_manager_mixin.rb

```diff
@@ -1,13 +1,12 @@
 module ManageIQ::Providers::Openstack::CinderManagerMixin
   extend ActiveSupport::Concern
-  include ::CinderManagerMixin
```

Comment:
I can vouch that most of this is not needed

do we need to define `ensure_cinder_manager`?
it is only called from one spot in `providers/openstack/cloud_manager.rb` -- so if you want to tweak this, it can be in this same PR

Reply:
Yeah so the `ManageIQ::Providers::Openstack::CloudManager` was overriding `ensure_cinder_manager` from the core `CinderManagerMixin` anyway, I didn't think it was necessary to pull it out of there and into this mixin when it was already with the rest of the `ensure_*_manager` methods.

Reply:
and the last one is good - carry on

Reply:
Good catch on the :type, updated https://github.com/ManageIQ/manageiq-providers-openstack/pull/677/files#diff-6e82d48dbcba5ccd5c9dee5a5f722c52485f2c738505fe1698d54d9786e69fa0L452